packages/launch-editor/editor-info/osx.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
   '/Applications/Visual Studio Code.app/Contents/MacOS/Electron': 'code',
   '/Applications/Visual Studio Code - Insiders.app/Contents/MacOS/Electron':
     'code-insiders',
+  '/Applications/VSCodium.app/Contents/MacOS/Electron': 'vscodium',
```

Comment:
The CLI command for VSCodium is `codium` now.
https://github.com/VSCodium/vscodium/pull/176
```suggestion
  '/Applications/VSCodium.app/Contents/MacOS/Electron': 'codium',
```